app/src/main/java/com/example/newsapp/ui/home/fragments/BottomSheetFragment.kt

```diff
@@ -0,0 +1,43 @@
+package com.example.newsapp.ui.home.fragments
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.example.newsapp.databinding.ImagePickerFragmentBinding
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+
+class BottomSheetFragment : BottomSheetDialogFragment() {
+    private var cameraListener: (()->Unit)? =null
```

Comment:
@jnandrawos please apply code formatting before pushing your changes for better readability

Reply:
done